data/supporters.json

```diff
@@ -1319,6 +1319,32 @@
           "name": "Daniela Bentrup",
           "twitter": "danibentrup",
           "email": "daniela@data-artisans.com"
+          }
```

Comment:
Could you fix the indentation (2 spaces for each step) so this diff looks nice? Line 1322 and 1335.

Thanks and welcome :)